Controllers/Functionalities/studentController.ts

```diff
@@ -0,0 +1,9 @@
+import { creationOfStudent } from "../../Models/INSERT";
+import { fetchStudents} from "../../Models/SELECT";
```

Comment:
no nee for functionality name. l functionality li hkine fya is for understanding the role of it. we rename the whole folder studentController. nside of it we have index.ts and studentController.ts
inside the index.ts you set the routing and inside studentController.ts you set where you get from the request body

Reply:
Done